src/Features/CSharp/Portable/DocumentationComments/CodeFixes/CSharpAddDocCommentParamNodeCodeFixProvider.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using System;
+
+namespace Microsoft.CodeAnalysis.DiagnosticComments.CodeFixes
+{
+    [ExportCodeFixProvider(LanguageNames.CSharp, Name = PredefinedCodeFixProviderNames.AddAttribute), Shared]
```

Comment:
Uh, thanks.  This started as a feature to add the name attribute to nodes without it, but I realized that we should probably just remove those since we can't add a specific name.